apex/ProductConfiguration.apex

```diff
@@ -80,23 +80,25 @@ insert omniscriptName;
 
 System.debug(omniscriptName);
 
-AttributePicklist attributePicklistLoanTerm;
+AttributePicklist attributePicklistLoanTerm =
+        new AttributePicklist();
 
 Schema.DescribeSObjectResult describeResult = Schema.getGlobalDescribe().get('AttributePicklist').getDescribe();
 Map<String, Schema.SObjectField> fieldsMap = describeResult.fields.getMap();
+Map<String, Object> fieldValues = new Map<String, Object>();
+fieldValues.put('Name', 'LoanTerm');
+fieldValues.put('Description', 'LoanTerm');
+fieldValues.put('DataType', 'Text');
+fieldValues.put('Status', 'Active');
 // FieldService perm is auto enabled in Developer Org which makes Code field visible and mandatory
 if (fieldsMap.containsKey('Code')) {
-    attributePicklistLoanTerm =
-        new AttributePicklist(Name = 'LoanTerm', Description='LoanTerm', Code = '1', DataType='Text', Status = 'Active');
-
-	insert attributePicklistLoanTerm;
+	fieldValues.put('Code', '1');
```

Comment:
Is this supposed to be
```
fieldValues.put('Code', fieldsMap.get('Code'));
```

Reply:
FieldsMap will not have the actual Code value '1' which we want to insert in fieldValues